src/docfx/template/TemplateEngine.cs

```diff
@@ -40,6 +42,20 @@ private TemplateEngine(string templateDir, JsonSchema metadataSchema)
                 .ToDictionary(prop => prop.Key, prop => prop.Value.HtmlMetaName);
         }
 
+        public static JsonSchema GetJsonSchema(Schema schema)
+        {
+            if (schema == null)
+            {
+                return null;
+            }
+
+            // TODO: get schema from template
```

Comment:
I guess test schemas should not go to Template?

Reply:
At that time, we should always have a test template for test purpose, just like legacy spec test do.